test/parallel/test-accessor-properties.js

```diff
@@ -33,27 +33,15 @@ const UDP = internalBinding('udp_wrap').UDP;
   }, TypeError);
 
   const StreamWrapProto = Object.getPrototypeOf(TTY.prototype);
+  const properties = ['bytesRead', 'fd', '_externalStream'];
 
-  // Should not throw for Object.getOwnPropertyDescriptor
-  assert.strictEqual(
-    typeof Object.getOwnPropertyDescriptor(StreamWrapProto, 'bytesRead'),
-    'object'
-  );
-
-  assert.strictEqual(
-    typeof Object.getOwnPropertyDescriptor(StreamWrapProto, 'fd'),
-    'object'
-  );
-
-  assert.strictEqual(
-    typeof Object.getOwnPropertyDescriptor(StreamWrapProto, '_externalStream'),
-    'object'
-  );
-
-  assert.strictEqual(
-    typeof Object.getOwnPropertyDescriptor(StreamWrapProto, 'fd'),
-    'object'
-  );
+  properties.forEach((property) => {
+    // Should not throw for Object.getOwnPropertyDescriptor
+    assert.strictEqual(
+      typeof Object.getOwnPropertyDescriptor(StreamWrapProto, property),
+      'object'
+    );
+  });
```

Comment:
While this change makes it easier for the reader, in its current form it makes it harder to diagnose a failure as the assertion failure will not indicate which property failed. Perhaps add a message argument to the `assert.strictEqual()` call indicating what has failed?

Reply:
resolved